RundeckAutomation/RundeckAutomation.psd1

```diff
@@ -18,7 +18,7 @@
     RootModule = 'RundeckAutomation.psm1'
 
     # Version number of this module.
-    ModuleVersion = '0.0.0'
+    ModuleVersion = '0.1.1'

```

Comment:
Set the version to `1.0.0` and update the whiskey.yml so you publish prerelease versions.